src/WebJobs.Script/Content/Script/http/response.js

```diff
@@ -7,6 +7,8 @@ module.exports = (context) => {
 
         end: (body) => {
             if (body !== undefined) {
+                if (Buffer.isBuffer(body) && !res.get('Content-Type'))
```

Comment:
I think the existing headers implementation has casing issues - I think it is case insensitive currently, when it should be case insensitive? Is Express response case insensitive? If so, we should fix and add tests.